src/main/java/core/basesyntax/service/strategy/Balance.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.service.ActivitiesService;
+
+public class Balance implements ActivitiesService {
+    @Override
+    public int quantityModify(int adderBefore) {
+        return adderBefore;
+    }

```

Comment:
what is the reason for this strategy? Just return the quantity that you put as a parameter? 
I'd recommend you to make an exact operation in every single operation, not return just an the same quantity